app/proc/processor.go

```diff
@@ -105,6 +112,15 @@ func (p *Processor) feed(name, url, telegramChannel string, max int) {
 			continue
 		}
 
+		skip, err := filter.skip(item)
```

Comment:
this part is confusing. The problem here is an implicit expectation of valid skip even if err != nil. This is unusual and even odd. I suggest adding to your `if skip` a condition `&& err ==nil`.